graphs/energy/nodes/buildings/buildings_space_heater_solar_thermal.ad

```diff
@@ -1,26 +1,28 @@
+- input.solar_thermal = 1.0
 - output.loss = elastic
-- output.useable_heat = 1.0
+- output.useable_heat = 0.48
 - groups = [heat_production, application_group, wacc_proven_tech, costs_building_and_installations_buildings]
 - use = energetic
 - presentation_group = traditional_heat
 - availability = 0.0
-- full_load_hours = 791.666666666667
+- full_load_hours = 600.0
 - ccs_investment = 0.0
 - construction_time = 0.0
 - cost_of_installing = 0.0
 - decommissioning_costs = 0.0
-- fixed_operation_and_maintenance_costs_per_year = 1837.5
-- initial_investment = 144375.0
-- technical_lifetime = 20.0
+- fixed_operation_and_maintenance_costs_per_year = 20.0
+- initial_investment = 600.0
+- technical_lifetime = 25.0
 - variable_operation_and_maintenance_costs_for_ccs_per_full_load_hour = 0.0
 - variable_operation_and_maintenance_costs_per_full_load_hour = 0.0
-- wacc = 0.04
+- wacc = 0.02
```

Comment:
If we check our documentation, we only use a 2% WACC for [households](https://docs.energytransitionmodel.com/main/cost-wacc#households-real-wacc-2), for heating technologies in the [services](https://docs.energytransitionmodel.com/main/cost-wacc#commercial--proven-technologies-real-wacc-4) sector a WACC of 4% is assumed.